src/TodoListItem.js

```diff
@@ -28,13 +50,13 @@ class TodoListItem extends React.Component {
   }
 
   render() {
-    let { content, priority } = this.props.item
+    let { task } = this.props.item
```

Comment:
All of your lets can be const. Always use const unless you can't for the sake of immutability.